lib/rubocop/cop/layout/space_inside_string_interpolation.rb

```diff
@@ -5,11 +5,18 @@ module Cop
     module Layout
       # This cop checks for whitespace within string interpolations.
       #
-      # @example
-      #   # Good if EnforcedStyle is no_space, bad if space.
+      # @example EnforcedStyle no_space (default)
```

Comment:
You forgot `:` here and in the other example.

Reply:
Oops. I fixed by #4994. Thanks for the review 💦